app/Commands/Command.php

```diff
@@ -63,11 +63,6 @@ abstract class Command extends BaseCommand
     /**
      * Creates a new command instance.
      *
-     * @param  \App\Repositories\ConfigRepository  $config
-     * @param  \App\Repositories\ForgeRepository  $forge
-     * @param  \App\Repositories\KeyRepository  $keys
-     * @param  \App\Repositories\RemoteRepository  $remote
-     * @param  \App\Support\Time  $time
      * @return void
```

Comment:
You can remove this one as well

Reply:
We are removing all `@return void` from constructors these days. You can remove it 👍